services/pagination.js

```diff
@@ -0,0 +1,25 @@
+const generateEllipsisBreaks = async (pages, currentPage) => {
```

Comment:
Why is this function async as it doesn't use promises?

```suggestion
const generateEllipsisBreaks = (pages, currentPage) => {
```

Reply:
I would question if pageList needs to be async as well as from what I see it is awaiting generateEllipsisBreaks as the function is set as async.

Maybe we can have a call and have a look together?

Reply:
sorry I misunderstoond your comments, update shortly.Thanks

Reply:
I have removed all the async for i have retested and it worked contrary to what happened during development.
So you are right, with what we've got it should work effectively without async.